lib/ex_dimensions/conversions/conversions.ex

```diff
@@ -7,9 +7,13 @@ defmodule ExDimensions.Conversions do
   require ExDimensions.Conversions.Graph
   @before_compile ExDimensions.Conversions.Graph
 
-  def %{value: v1, units: u, denom: []} ~> u2 when is_list(u2) do
-    {:ok, value} = convert(v1, u, u2)
-    %{value: value, units: u2, denom: []}
+  alias ExDimensions.Quantity
+
+  def %Quantity{value: v1, units: u, denom: []} ~> u2 when is_list(u2) do
+    case convert(v1, u, u2) do
```

Comment:
I changed this block to use the same `case` logic as the other function definition